docs/source/API/core/Macros.rst

```diff
@@ -4,11 +4,71 @@ Macros
 Version Macros
 --------------
 
-+--------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| Macro              | Description                                                                                                                                                                  |
-+====================+==============================================================================================================================================================================+
-| ``KOKKOS_VERSION`` | The Kokkos version; ``KOKKOS_VERSION % 100`` is the patch level, ``KOKKOS_VERSION / 100 % 100`` is the minor version, and ``KOKKOS_VERSION / 10000`` is the major version.   |
-+--------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+.. list-table::
+   :header-rows: 1
+
+   * - Version integer macros
+   * - ``KOKKOS_VERSION``
+   * - ``KOKKOS_VERSION_MAJOR``
+   * - ``KOKKOS_VERSION_MINOR``
+   * - ``KOKKOS_VERSION_PATCH``
```

Comment:
I don't like the looks of the one column tables very much and would either replace them with a list or add a second column that demonstrates their definition for some release.